service/src/main/java/com/ge/predix/acs/audit/PredixEventProcessor.java

```diff
@@ -0,0 +1,34 @@
+package com.ge.predix.acs.audit;
+
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.ge.predix.audit.AuditEvent;
+import com.ge.predix.audit.AuditEventProcessor;
+import com.ge.predix.audit.sdk.AuditClient;
+import com.ge.predix.audit.sdk.exception.AuditException;
+import com.ge.predix.audit.sdk.message.AuditEventV2;
+
+public class PredixEventProcessor implements AuditEventProcessor {
+
+    private static final org.slf4j.Logger LOGGER = LoggerFactory.getLogger(PredixEventProcessor.class);
+
+    @Autowired
+    private AuditClient auditClient;
+
+    @Autowired
+    private PredixEventMapper eventMapper;
+
+    @Override
+    public boolean process(final AuditEvent auditEvent) {
+        AuditEventV2 predixEvent = eventMapper.map(auditEvent);
+        try {
```

Comment:
Boolean is required because of the way we created the tests for the AuditEventProcessor. Also we shouldn't add the throws at the interface level because difference implementations of  AuditEventProcessor may not use the Predix-Audit SDK.